docs(fix): wording MTA-1561 (#202)
scaleway/docs-content: tutorials/remote-desktop-with-xrdp/index.mdx

- You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
- You have [configured your SSH Key](/console/my-project/how-to/create-ssh-key/)

> - You have a Virtual Instance running on Ubuntu Bionic Beaver (18.04).

Reviewer: ```suggestion
- You have an Instance running on Ubuntu Bionic Beaver (18.04).
```
